src/math/ColorManagement.js

```diff
@@ -14,14 +13,13 @@ export function LinearToSRGB( c ) {
 
 }
 
-
 /**
  * Matrices converting P3 <-> Rec. 709 primaries, without gamut mapping
  * or clipping. Based on W3C specifications for sRGB and Display P3,
  * and ICC specifications for the D50 connection space. Values in/out
  * are _linear_ sRGB and _linear_ Display P3.
  *
- * Note that both sRGB and Display P3 use the sRGB transfer functions.
+ * Note that both sRGB and Display P3 use the sRGB transfer function.
```

Comment:
nit: this was intentional, I believe it's standard to refer to transfer functions (to/from X) as plural. I've tried to be consistent about that [in the docs](https://threejs.org/docs/#manual/en/introduction/Color-management) as well.

Reply:
I think there is only one transfer function. The other is the inverse transfer function.

Reply:
I think there is only one transfer function. The other is the inverse transfer function.

But I'll change it.